lib/ruby_llm.rb

```diff
@@ -30,8 +30,8 @@ module RubyLLM
   class Error < StandardError; end
 
   class << self
-    def chat(model: nil, provider: nil)
-      Chat.new(model: model, provider: provider)
+    def chat(model: nil, provider: nil, max_tool_completions: config.max_tool_completions)
+      Chat.new(model: model, provider: provider, max_tool_completions: max_tool_completions)
```

Comment:
changing the whole interface to `chat` is a bit heavy handed. this should be a simple config change.

Reply:
@crmne would love an update here ... I'm getting looping tool calls as well, and would like to avoid implementing my own solution.